### PR TITLE
Update Maschinendeck endpoint

### PR DESCRIPTION
**Pull request by miko007**

### CODE REVIEW

**Comment by dbrgn** [comment]
The SVG will cause issues with clients that cannot handle it (e.g. MyHackerspace on Android), but we don't currently enforce any guidelines regarding file formats.

However, since HTTPS is available, could you maybe change the URL to use HTTPS?

File: directory.json [hunk 1]
```diff
@@ -105,7 +105,7 @@
   "Makerspace Erfurt":"https://status.makerspace-erfurt.de/status.json",
   "Makerspace Gütersloh":"https://makerspace-gt.de/space-api/space-api.json",
   "MakeSpace Madrid" : "https://makespacemadrid.org/spaceapi/status.json",
-  "Maschinendeck":"http://state.maschinendeck.org/spaceapi.json",
+  "Maschinendeck":"http://maschinenstate.42dots.de/",
```
Comment (rnestler) [inline directory.json]: You could change it to `https` since it seems to work (but not forward).

**Comment by miko007** [comment]
> The SVG will cause issues with clients that cannot handle it (e.g. MyHackerspace on Android), but we don't currently enforce any guidelines regarding file formats.

That then is a flaw in the mentioned apps. SVG is the de facto standard for small filesize - high resolution images on the internet and by many means the superior format. I think you should file an issue in the affected projects, then.

> However, since HTTPS is available, could you maybe change the URL to use HTTPS?

My bad, i did not check when i copy & pasted the url. i will correct that.

**Comment by dbrgn** [comment]
> That then is a flaw in the mentioned apps.

Correct SVG parsing is *highly* complex. Even among web browsers, the degree of support varies, with no browser supporting the entire feature set: https://en.wikipedia.org/wiki/Comparison_of_layout_engines_%28Scalable_Vector_Graphics%29

Android as an app platform does not provide a method to render SVGs. The "MyHackerspace" app is currently 103 KiB in size. Adding SVG support would mean adding a multi-megabyte library to render logos.

I'm a big fan of SVG, but if you want to maximize compatibility across multiple platforms, then pixel formats are still superior. However, as long as we haven't specified any constraints regarding the file formats, continuing to use SVG is perfectly fine.

(Maybe we should add support for multi-format logos to the spec...)

**Comment by miko007** [comment]
> (Maybe we should add support for multi-format logos to the spec...)

yep, that seems like the way to go, then.

**Comment by miko007** [comment]
all issues mentioned here have been resolved. i will open a new pull request now. thank you all for your input.

**Pull request closed without merging**